packages/jaeger-ui/src/utils/redux-form-field-adapter.js

```diff
@@ -14,22 +14,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { Popover } from 'antd';
+import cx from 'classnames';
 import * as React from 'react';
 
+import './redux-form-field-adapter.css';
+
 export default function reduxFormFieldAdapter(
   AntInputComponent: Class<React.Component<*, *>>,
   onChangeAdapter: () => void
 ) {
   return function _reduxFormFieldAdapter(props: any) {
-    const { input: { value, onChange }, children, ...rest } = props;
+    // inputRest includes necessary props such as onBlur and value
+    const { input: { onChange, ...inputRest }, children, ...rest } = props;
+    const isInvalid = rest.meta.touched && !!rest.meta.error;
     return (
-      <AntInputComponent
-        value={value}
-        onChange={onChangeAdapter ? (...args) => onChange(onChangeAdapter(...args)) : onChange}
-        {...rest}
-      >
-        {children}
-      </AntInputComponent>
+      <Popover placement={'bottomLeft'} visible={isInvalid} {...rest.meta.error}>
```

Comment:
Maybe a parameter can be added to `reduxFormFieldAdapter` for whether or not the `<Popover />` should be used? When custom time-range is selected, six input fields are not using it but it's still be rendered.

```js
export default function reduxFormFieldAdapter(
   AntInputComponent: Class<React.Component<*, *>>,
   onChangeAdapter: () => void,
   isValidatedInput?: boolean,
 ) {
```

Reply:
added a parameter which also fixed an issue with the dropdown